components/Chat/AutocompleteChat.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <v-sheet
-    class="chat-autocomplete body-2 py-1"
+    class="chat-autocomplete body-2 pl-1 ml-3 py-1"
```

Comment:
it kinda looks like you might be adding extra padding and margin to compensate for the wrong translated value below.

Reply:
removed